### PR TITLE
Audio atom as an island

### PR DESCRIPTION
**Pull request by OllysCoding**
Adds audio atom as an island

Dependant on Frontend passing `switches` to /Blocks request (https://github.com/guardian/frontend/pull/24754)

### CODE REVIEW

**Comment by OllysCoding** [comment]
[AudioAtom](https://trello.com/c/zwYy8sf2/623-audioatom)

File: dotcom-rendering/index.d.ts [hunk 1]
```diff
@@ -579,6 +579,7 @@ interface BlocksRequest {
 	sharedAdTargeting: Record<string, unknown>;
 	adUnit: string;
 	videoDuration?: number;
+	switches: { [key: string]: boolean };
```
Comment (OllysCoding) [inline dotcom-rendering/index.d.ts:582]: See https://github.com/guardian/frontend/pull/24754

File: dotcom-rendering/src/web/lib/renderElement.tsx [hunk 2]
```diff
@@ -127,20 +130,28 @@ export const renderElement = ({
...
+					<AudioAtomWrapper
+						id={element.id}
+						trackUrl={element.trackUrl}
+						kicker={element.kicker}
+						title={element.title}
+						duration={element.duration}
+						pillar={format.theme}
+						contentIsNotSensitive={isSensitive}
+						aCastisEnabled={switches.acast}
+						readerCanBeShownAds={isAdFreeUser}
```
Comment (AshCorr) [inline dotcom-rendering/src/web/lib/renderElement.tsx:152]: should this be `!isAdFreeUser` ?

File: dotcom-rendering/src/web/lib/renderElement.tsx [hunk 3]
```diff
@@ -127,20 +130,28 @@ export const renderElement = ({
...
-				/>,
+				<Island>
+					<AudioAtomWrapper
+						id={element.id}
+						trackUrl={element.trackUrl}
+						kicker={element.kicker}
+						title={element.title}
+						duration={element.duration}
+						pillar={format.theme}
+						contentIsNotSensitive={isSensitive}
```
Comment (AshCorr) [inline dotcom-rendering/src/web/lib/renderElement.tsx:150]: should this be `!isSensitive` ?

**Review by oliverlloyd — APPROVED** [review]

**Comment by mxdvl** [comment]
🏝️  #3629

**Pull request merged**